website/content/api-docs/agent/check.mdx

```diff
@@ -206,6 +206,9 @@ The table below shows this endpoint's support for
 
 - `Body` `(string: "")` - Specifies a body that should be sent with `HTTP` checks.
 
+- `DisableRedirects` `(bool: false)` - Specifies whether to disable following HTTP
```

Comment:
Do we want caveat about the future change in the default here as well?

Reply:
If our intent is to change the default later (which would be a breaking change for health checks that relied on redirects) shouldn't we document that beforehand? Or is a changelog note enough?

Reply:
It might be better to state this in the 1.12 release notes. When the default value is changed in a later release, it should be called out as a breaking change in the changelog, and also mentioned in the upgrade guide.

Reply:
Ok - it's in the changelog but I removed both occurrences of the note from the docs